content/enterprise_influxdb/v1.8/administration/config-meta-nodes.md

```diff
@@ -66,10 +65,7 @@ Use the same key for all nodes in the same cluster.
 {{% warn %}}The `license-key` and `license-path` settings are mutually exclusive and one must remain set to the empty string.
 {{% /warn %}}
 
-InfluxData recommends performing rolling restarts on the nodes after the license key update.
-Restart one meta node or data node service at a time and wait for it to come back up successfully.
-The cluster should remain unaffected as long as only one node is restarting at a
-time as long as there are two or more data nodes.
+> **Note:** See how to [renew or update your license key](/enterprise_influxdb/v1.8/administration/renew-license/).
```

Comment:
My understanding is that it impacts the license key and/or license file.  Use cases are:
- switching from key to file (more common), (or from file to key...but this is rare)
- changing the key itself (from one key to another)
- changing the license file (either by replacing the existing file that the configuration points to on disk or by changing where the configuration file is within the file -- could location on disk or file name, etc.)

Reply:
meta nodes must be restarted to re-load their config.... but, now data nodes can reload when sent the sighup signal

Reply:
> My understanding is that it impacts the license key and/or license file. Use cases are:
> 
> * switching from key to file (more common), (or from file to key...but this is rare)
> * changing the key itself (from one key to another)
> * changing the license file (either by replacing the existing file that the configuration points to on disk or by changing where the configuration file is within the file -- could location on disk or file name, etc.)

@timhallinflux, added an intro sentence to cover use cases, and a step to delete existing license if they're switching. Do you think we need more detail about the different ways of changing the license file?

Reply:
> meta nodes _must be_ restarted to re-load their config.... but, now data nodes can reload when sent the sighup signal

Added note about restarting meta nodes and reloading data nodes in the respective Enterprise Settings configurations